Change the SRC_URI for AudioManager

DunderRoffe opened the pull request:
Change the SRC_URI for AudioManager from "git.projects.genivi.org" to
"github.com".

Add SRCREV since branch tracking is dangerous and can not be used offline.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

chbae commented on meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:
```diff
@@ -9,8 +9,9 @@ DEPENDS = "qtbase qtdeclarative qtwebkit"
 BRANCH = "Intreprid_stable_branch"
 VERSION = "6.1"
```
If you use SRCREV, you don't need to BRANCH and VERSION variables. Please remove those.

zeenix replied:
But he is no longer using SRCREV (yet).

chbae commented on meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:
```diff
@@ -9,8 +9,9 @@ DEPENDS = "qtbase qtdeclarative qtwebkit"
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
 SRC_URI = "\
-    git://git.projects.genivi.org/AudioManager.git;branch=${BRANCH};tag=${VERSION};protocol=http \
+    git://github.com.genivi/AudioManager.git;protocol=http \
```
git://github.com/GENIVI/AudioManager.git;protocol=http

chbae commented:
@DunderRoffe 

I agree with your scheme but you need to change somethings.

DunderRoffe commented:
Good catch :) Now it is fixed.

zeenix reviewed (changes requested):
"Change the SRC_URI for AudioManager"

Is not specific enough to be very helpful, "Change" being a very vague term. Let's be a bit more specific about the change.

Also you want to add "audiomanager: " prefix to commit (and hence "for AudioManager" becomes redundant).

zeenix commented on meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb:
```diff
@@ -6,11 +6,9 @@ LIC_FILES_CHKSUM = "file://AudioManagerPoC/README;md5=be0fe05c89ff7a2632eb080f72
-BRANCH = "Intreprid_stable_branch"
-VERSION = "6.1"
-
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
```
One more thing: This is a separate change, hence should be in a separate commit.

DunderRoffe replied:
Then I would have to rewrite the other part making an unnecessary step adding the branch stuff to the other URI. I get the point of keeping separate commits but this seems more weird than helpful.

zeenix replied:
> Then I would have to rewrite the other part 

Yes, that's true but now this will also force you to get into always thinking about dividing your changes from the very beginning and plan better.

> I get the point of keeping separate commits but this seems more weird than helpful.

You either get it or you don't. :) We need to have same rules for everyone and every commit so in this case it might not be very helpful to divide but once you start bending the rules, where do you draw the line?

zeenix replied:
Oh and summary/shortlog can only describe one change so if you have any other changes (no matter how small) in your commit, they'll be missed when someone is reading through shortlog of changes.

zeenix reviewed (changes requested):
"Redefine the SRC_URI for AudioManager from "git.projects.genivi.org" to "github.com"."

This is not bad summary at all so this is just an improvement idea: I tend to try and avoid mentioning implementation details like "changing of SRC_URI" in commit log. So in here I'd have wrote:

"Fetch AudioManager from github.com instead of git.projects.genivi.org because [reasons]".

Commit:
AudioManager: Use github.com as SRC_URI

Redefine the SRC_URI for AudioManager from "git.projects.genivi.org" to
"github.com".

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

Commit:
AudioManager: Use SRCREV instead of tag tracking

Replace branch and index tracking with specific SRCREV. This is important
since branch tracking is dangerous and can not be used offline.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>
